tests1/myfile2.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+echo $_POST[ 'phpcs should catch this problem' ];
```

Comment:
:warning: **Warning**: Detected usage of a non-sanitized input variable: $_POST['phpcs should catch this problem'] (*WordPress.Security.ValidatedSanitizedInput.InputNotSanitized*).